python-mini-projects/projects/Web_scraping_a_youtube_comment/webscrapindcomment.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 21 16:55:39 2020
+
+@author: hp
+"""
+
+from selenium import webdriver
+import csv
+import time
+
+items=[]
+driver=webdriver.Chrome(r"C:/Users/hp/Anaconda3/chromedriver.exe")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *driver*. The resource is allocated by call *webdriver.Chrome*. Execution paths that do not contain closure statements were detected. To prevent this resource leak, close *driver* in a try-finally block or declare it using a `with` statement.

**More info**
[View details about the `with` statement in the Python developer&#39;s guide](https://www.python.org/dev/peps/pep-0343/) (external link).